helm/charts/kratos/templates/statefulset-mail.yaml

```diff
@@ -133,3 +133,7 @@ spec:
             name: {{ include "kratos.fullname" $root }}-template-{{ $method }}-{{ $result }}
         {{- end }}
         {{- end }}
+      {{- with .Values.nodeSelector }}
```

Comment:
Please add a ternary operation to the selector like with 
`{{- $extraVolumes := ternary .Values.statefulSet.extraVolumes .Values.deployment.extraVolumes (not (empty .Values.statefulSet.extraVolumes )) -}}`
To allow setting it for the statefulset and deployment separately

Reply:
ok i see, issue is there is a `.Values.nodeSelector` and a `.Values.deployment.nodeSelector`, and only the `.Values.nodeSelector` was used. I did something, tell me what you think about it.